src/shared/icons/Logo.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+// TODO: add props for controlling color and size
+const Logo = () => {
+  return (
+    <>
+      {/* <!-- Uploaded to: SVG Repo, www.svgrepo.com, Generator: SVG Repo Mixer Tools --> */}
```

Comment:
TODO: You can move this comment to the top of the file (above the import statement), this is standard for copyright shit and comments for the whole file.

Reply:
Done. I will create a PR now.